.github/workflows/py_checks.yml

```diff
@@ -12,7 +12,7 @@ on:
       - 'samples/python/**'
 jobs:
   linters:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-18.04
```

Comment:
Why did you reduce the versuion of OS?

Reply:
to wait for full python 3.10 support

Reply:
Just an idea: can we switch it to something more updated? Like Ubuntu20 and Python3.7/3.8/3.9 for example? Because this older version won't probably get good updates on packages that are used later in py_checks.

Reply:
Sure, I've tried this in a different PR, but for some reason python checks were not triggered. But in other PRs on master with U22 and python 3.10 python checks started to fail (example https://github.com/openvinotoolkit/openvino/pull/12530).
So, I suppose we need to enable python 3.10 first and after revert change in this file